src/bitmap.h

```diff
@@ -578,4 +571,20 @@ class Bitmap {
 	bool read_only = false;
 };
 
+inline ImageOpacity Bitmap::GetImageOpacity() const {
+	return image_opacity;
+}
+
+inline ImageOpacity Bitmap::GetTileOpacity(int row, int col) const {
+	return !tile_opacity.Empty() ? tile_opacity.Get(row, col) : ImageOpacity::Partial;
```

Comment:
Why is it here Get(row, col)

Reply:
Typo: this should be `x` and `y` to match the new `TIleOpacity` class.